OWNERS_ALIASES

```diff
@@ -222,6 +222,7 @@ aliases:
     - krzyzacy
```

Comment:
we should probably prune this list, in particular @krzyzacy @ixdy

Reply:
feel free to remove me :-)

Reply:
will do in another PR @krzyzacy

Reply:
I sent an inactive-googler prune to test-infra, will prune k/k next

Reply:
https://github.com/kubernetes/kubernetes/pull/93980 will probably wind up rebasing ontop of this one later